content/developer/reference/frontend/odoo_editor.rst

```diff
@@ -0,0 +1,120 @@
+
+.. highlight:: javascript
+
+.. default-domain:: js
+
+===========
+Odoo Editor
+===========
+
+Odoo Editor is Odoo's own rich text editor. It has replaced the Summernote
+library since Odoo 15.0. It can be found at
+*web_editor/static/src/js/editor/odoo-editor*.
+
+Full documentation will come soon.
+
+Tests
+=====
+
+The editor comes with its own series of unit tests that can be seen at the route
+*/web_editor/tests*.
+
+Powerbox
+========
+
+The Powerbox is a piece of user interface that contains *commands* organized
+into *categories*. The commands can be filtered when the user inputs text, and
+navigated with the arrow keys.
+
+.. image:: editor/powerbox.png
```

Comment:
Please compress the image with [pngquant](https://pngquant.org/).
It's not incredibly useful here, but it's a good habit to take since we'll soon test that images are indeed compressed.